config/covid-19/app_settings.json

```diff
@@ -163,9 +163,7 @@
     ]
   },
   "changes_controller": {
-    "reiterate_changes": true,
-    "changes_limit": 100,
-    "debounce_interval": 200
+    "changes_limit": 100
   },
```

Comment:
Let's just remove this `changes_controller` property altogether, seeing as it's using the default anyway.